jenkins_jobs/upload_iso/script.sh

```diff
@@ -0,0 +1,20 @@
+BUILD_DIR_PATH="${UPLOAD_SERVER_BUILDS_DIR}/${UPLOAD_SERVER_BUILD_ID}"
+BUILD_DIR_RSYNC_URL="${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${BUILD_DIR_PATH}"
+ISO_DIR_RSYNC_URL="${BUILD_DIR_RSYNC_URL}/iso"
+
+# Create remote iso directory
+mkdir iso
+rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
+      --recursive --verbose --compress --stats --times --chmod=a+rwx,g+rwx,o- \
+      iso \
+      "${BUILD_DIR_RSYNC_URL}"
+
+rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
+      --verbose --compress --stats --times --chmod=a+rwx,g+rwx,o- \
+      *.iso \
+      "${ISO_DIR_RSYNC_URL}/"
+
+rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
+      --verbose --compress --recursive --stats --times --chmod=a+rwx,g+rwx,o- \
+      *-CHECKSUM \
+      "${ISO_DIR_RSYNC_URL}/"
```

Comment:
What about defining an alias for rsync command with common parameters, for example:

```shell
alias rsync="rsync -e 'ssh -i ${HOME}/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa' \
    --verbose --compress --stats --times --chmod=a+rwx,g+wx,o-"

rsync iso ${BUILD_DIR_RSYNC_URL}  # push empty directory
rsync *.iso ${ISO_DIR_RSYNC_URL}  # push iso image
rsync *-CHECKSUM ${ISO_DIR_RSYNC_URL}  # push checksum file
```

I think you do not need to add trailing `/` in the last two rsync commands.